### PR TITLE
Block traffic protection

### PR DESCRIPTION
**Pull request by maybegreat48**

### CODE REVIEW

File: src/hooks/protections/received_event.cpp [hunk 1]
```diff
@@ -564,6 +564,11 @@ namespace big
 				g_pointers->m_gta.m_send_event_ack(event_manager, source_player, target_player, event_index, event_handled_bitset);
 				return;
 			}
+			else if (type == WorldStateDataType::PopMultiplierArea && g.protections.stop_traffic) // never used for legitimate reasons
```
Comment (Yimura) [inline src/hooks/protections/received_event.cpp]: Not even in races where traffic may end up being modified?

Comment (gir489returns) [reply src/hooks/protections/received_event.cpp:567]: Races usually happen on areas that are like under the world like -8000 X -8000 Y or something like that IIRC.

Comment (SkiddyToast) [reply src/hooks/protections/received_event.cpp]: This is used legitimately.
image

Comment (SkiddyToast) [reply src/hooks/protections/received_event.cpp]: > Races usually happen on areas that are like under the world like -8000 X -8000 Y or something like that IIRC.

It modifies the entire map at times.
image

Comment (gir489returns) [reply src/hooks/protections/received_event.cpp:567]: @SkiddyToast Do you know what circumstances the freemode script calls this function?

Comment (maybegreat48) [reply src/hooks/protections/received_event.cpp]: @SkiddyToast good catch, I've assumed all of the native calls had the local parameter set to true. This one applies it across all machines
Freemode always calls it with localOnly enabled, so we don't have to worry about that

**Review by Yimura — APPROVED** [review]

**Pull request merged**